ag_data/models.py

```diff
@@ -53,6 +53,9 @@ class AGSensor(models.Model):
     name = models.CharField(max_length=1024, blank=True)
     type_id = models.ForeignKey(AGSensorType, null=False, on_delete=models.PROTECT)
 
+    # def __str__(self):
+    #    return u"{0}".format(self.name)
+
```

Comment:
Optional: I guess we'd better delete commented lines because there's no point of maintaining it in the codebase but let's revisit it later.

Reply:
good point - we will remove this